home.html

```diff
@@ -0,0 +1,190 @@
+                    <li>
+                        <p class="sub-list-text">
+                            Immerse yourself in a world of artistic wonder as you witness a remarkable 
+                            showcase of artworks that embody the fusion of creative disciplines.
+                        </p>
+                    </li>
+                </ul>
+            </li>
+            <li class="list-item">
+                <ul class="sub-list">
+                    <li><img src="icons/program_icon_03.png" alt="program icon"></li>
+                    <li>
+                        <p class="sub-list-title">Forum</p>
+                    </li>
+                    <li>
+                        <p class="sub-list-text">
+                            Participants will have the opportunity to actively contribute to the discussions, 
+                            share their perspectives, ask questions, and gain insights from the diverse range of voices in attendance.
+                        </p>
+                    </li>
+                </ul>
+            </li>
+            <li class="list-item">
+                <ul class="sub-list">
+                    <li><img src="icons/program_icon_04.png" alt="program icon"></li>
+                    <li>
+                        <p class="sub-list-title">Workshop</p>
+                    </li>
+                    <li>
+                        <p class="sub-list-text">
+                            The workshop sessions provide a unique opportunity about practical aspects of interdisciplinary art and gain valuable insights from experts in their respective fields.
+                        </p>
+                    </li>
+                </ul>
+            </li>
+            <li class="list-item">
+                <ul class="sub-list">
+                    <li><img src="icons/program_icon_05.png" alt="program icon"></li>
+                    <li>
+                        <p class="sub-list-title">Network</p>
+                    </li>
+                    <li>
+                        <p class="sub-list-text">
+                            This is the perfect setting to expand your horizons and establish connections that can enrich your creative endeavors.
+                        </p>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+        <div class="btn-submit-container">
+            <button class="btn-submit" type="submit">Join Art Exibition 2023</button>
+        </div>
+        <a id="seeMore" href="#">See the whole program</a>
+    </section>
+    <section class="speaker-section">
+        <div class="speaker-intro-container">
+            <h3 class="speaker-intro-text">Featured speakers</h3>
+            <div class="guide-bar"></div>
+        </div>
+        <div id="speakers">
+            
+        </div>
+        <button id ="see-more-btn" type="submit">MORE<img src="icons/arrow_down.png" alt="Arrow down image"></button>
+    </section>
```

Comment:
## Featured Speakers
- in this section, you should have some margin on each side of the container to make to design more compact and tidy
- please reduce the size of the images because the larger the image the lower the quality and resolution and please make the height and weight of all images consistent with each other
- as seen on the template design you should have a checker-like design behind each image please add that

your design 
![image](https://github.com/Pi1998/first-capstone/assets/60612329/801b8162-ab6c-4d7f-9924-be85e4fe3164)

template design
![image](https://github.com/Pi1998/first-capstone/assets/60612329/943b271e-6572-4ebe-99df-9de633b2398e)